src/components/Fee/FeePage.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <el-container>
+    <el-main>
+      <el-row>
+        <el-col :xs="24" :lg="{ span: 18, offset: 3 }" :xl="{ span: 16, offset: 4 }">
+          <el-card :body-style="{ padding: '0' }">
+            <div class="header">
+              <span>Fee setting</span>
+            </div>
+            <el-table
+              :data="availableAssets.filter(data => !search || data.name.toLowerCase().includes(search.toLowerCase()))"
```

Comment:
How to fill in `search` variable? I did not find any input that connected to `search` variable.
![image](https://user-images.githubusercontent.com/16295803/58012166-8ac4a080-7afc-11e9-8c49-f2c0e4072291.png)